worlds/rogue_legacy/Regions.py

```diff
@@ -90,7 +85,7 @@ def create_regions(multiworld: MultiWorld, player: int):
 
     # Set up the regions correctly.
     for name, data in regions.items():
```

Comment:
this would probably be cleaner as
 ```
for region in regions:
  multiworld.regions.append(create_region(World, region)
  ```

Reply:
Rather it broken up here rather than at the start of `create_region`.